nextjs/package.json

```diff
@@ -8,12 +8,12 @@
     "start": "PORT=3001 next start"
   },
   "dependencies": {
-    "@types/node": "18.15.10",
-    "@types/react": "18.0.30",
-    "@types/react-dom": "18.0.11",
-    "next": "13.2.4",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "typescript": "5.0.2"
+    "@types/node": "^22.0.0",
```

Comment:
Maybe if we are setting the version of the types to v22 then we should enforce a node version in the package.json as well:
```
"engines": {
    "node": ">=22.0.0"
  },
```

Reply:
Done